### PR TITLE
Fix broken link in winui3_migration.md

### PR DESCRIPTION
**Pull request by krschau**
Fix broken link to try-convert documentation.

### CODE REVIEW

File: docs/winui3_migration.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ This porting assistance is provided in the form of Roslyn Analyzers and Code Fix
 - A Code Analyzer provides on the fly code inspections for C# and creates diagnostics. 
 - A Code Fix consumes the diagnostics created by the Code Analyzer and modifies the C# file in-line. 
 
-See the [try-convert documentation](https://github.com/dotnet/try-convert/blob/feature/winui) for more information on these code analyzers and how to automoate the process with [try-convert](https://github.com/dotnet/try-convert/blob/feature/winui/WinUIConvert.md).
\ No newline at end of file
+See the [try-convert documentation](https://github.com/dotnet/try-convert/blob/feature/winui/WinUIConvert.md) for more information on these code analyzers and how to automoate the process with try-convert.
```
Comment (roxk) [inline docs/winui3_migration.md:21]: The link in "It is recommended to use the try-convert tool for the full conversion process. try-convert will automate most of the conversion steps.", i.e. https://github.com/dotnet/try-convert/blob/feature/winui is still broken in the beginning of the file.

Comment (krschau) [reply docs/winui3_migration.md:21]: Thanks for the catch! I think all the links are good now.

**Review by roxk — APPROVED** [review]

File: docs/winui3_migration.md (hunk 1 above)
Comment (asklar) [reply docs/winui3_migration.md:21]: See #6391

**Review by JesseCol — APPROVED** [review]

**Pull request merged**